examples/disaggregated/slurm/disagg_profiler/job_manager.py

```
@@ -0,0 +1,1033 @@
+        ]
+        if 'ignore_eos' in self.config['profile'] and self.config['profile'][
+                'ignore_eos']:
+            cmd += ["--ignore-eos"]
+        cmd += ["&> " + os.path.join(output_folder, "benchmark_serving.log")]
+        print(f"Running command: {' '.join(cmd)}")
+        return self.spawn_in_trtllm_container(cmd, hostname)
+
+    def terminate_jobs(self, jobs):
+        """Terminate all running jobs."""
+        for job in jobs:
+            if job and job.poll() is None:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)